commands/utility/remindme.js

```diff
@@ -0,0 +1,30 @@
+exports.run = async (handler, message, args, pre) => {
+	if (!args.hours && !args.minutes && !args.seconds) {
+		return message.channel.send('No arguments given.')
```

Comment:
This will return "No arguments given." even if you do give a reason argument. Please make sure the message accurately reflects which arguments are required.